### PR TITLE
#39 form page chief complaint section

### PR DESCRIPTION
**Pull request by Wannakorn-C**
### Short Description

- Create chief complaint section
### Example UI
Fern

Screen Shot 2564-07-21 at 23 49 36

Figma

Screen Shot 2564-07-21 at 23 53 33

### CODE REVIEW

File: src/Pages/Form/Components/ChiefComplaintSection.tsx [hunk 1]
```diff
@@ -0,0 +1,207 @@
...
+  const handleClickGradual = () => {
+    setGradual(!Gradual);
+  };
+  //Rapidly
+  const [Rapidly, setRapidly] = React.useState(true);
+  const handleClickRapidly = () => {
+    setRapidly(!Rapidly);
+  };
+  //checkbox
+  const [state, setState] = React.useState({});
```
Comment (palmmy2542) [inline src/Pages/Form/Components/ChiefComplaintSection.tsx]: Please change state name to provide meaning(eg. symptomsCheckbox)

File: src/Pages/Form/Components/ChiefComplaintSection.tsx [hunk 2]
```diff
@@ -0,0 +1,207 @@
...
+    secondary: {
+      main: "#EF5DA8",
+      contrastText: "#fff",
+    },
+  },
+});
+
+const ChiefComplaintSection: React.FC = () => {
+  const classes = useStyle();
+  //WakeUp
```
Comment (palmmy2542) [inline src/Pages/Form/Components/ChiefComplaintSection.tsx]: It should be grouping all of button states (eg. timeCourse) to object state as a checkboxes state.

Comment (palmmy2542) [reply src/Pages/Form/Components/ChiefComplaintSection.tsx]: And, Can it be selected more than one? Please ask PM(P' Pee) to clarify.

File: src/Pages/Form/Components/ChiefComplaintSection.tsx [hunk 3]
```diff
@@ -0,0 +1,207 @@
...
+      <Box>
+        <Typography className={classes.text} variant="body1">
+          Symptoms
+        </Typography>
+        <FormGroup>
+          <FormControlLabel
+            control={
+              <PinkCheckbox
+                onChange={handleChange}
+                name="Alteration of consciousness"
```
Comment (palmmy2542) [inline src/Pages/Form/Components/ChiefComplaintSection.tsx]: I suggest defining name as a camel case(eg. alterationOfConsciousness)

Comment (palmmy2542) [reply src/Pages/Form/Components/ChiefComplaintSection.tsx]: Except for label, it is ok.

File: src/Pages/Form/Components/SectionTitle.tsx [hunk 4]
```diff
@@ -8,7 +8,7 @@ const useStyle = makeStyles((theme: Theme) =>
   createStyles({
     root: {
       width: "100%",
-      padding: "118px",
+      padding: "50px",
```
Comment (palmmy2542) [inline src/Pages/Form/Components/SectionTitle.tsx]: Why did you change padding of SectionTitle?

File: src/Pages/Form/Components/ChiefComplaintSection.tsx [hunk 5]
```diff
@@ -0,0 +1,207 @@
...
+              />
+            }
+            label="Alteration of consciousness"
+          />
+          <FormControlLabel
```
Comment (palmmy2542) [inline src/Pages/Form/Components/ChiefComplaintSection.tsx]: FormControllabel can be grouped to Array and use map function to render a component.. Because it has a property and design similarly.

Comment (palmmy2542) [reply src/Pages/Form/Components/ChiefComplaintSection.tsx]: - [ ] Change FormControlLabel to Array and use map function to render them.

File: src/Pages/Form/Components/ChiefComplaintSection.tsx [hunk 6]
```diff
@@ -0,0 +1,207 @@
...
+  return (
+    <Box className={classes.root}>
+      <ThemeProvider theme={theme}>
+        {/* Time Course */}
+        <Box>
+          <Typography className={classes.text} variant="body1">
+            Time Course
+          </Typography>
+          <Box className={classes.groupButton}>
+            <Button
```
Comment (palmmy2542) [inline src/Pages/Form/Components/ChiefComplaintSection.tsx]: Button can be groped. (The reason similar a line 155)

File: src/Pages/Form/Components/ChiefComplaintSection.tsx [hunk 7]
```diff
@@ -0,0 +1,207 @@
...
+import Button from "@material-ui/core/Button";
+import {
+  createStyles,
+  makeStyles,
+  createTheme,
+  Theme,
+  ThemeProvider,
+  withStyles,
+} from "@material-ui/core/styles";
+import {
```
Comment (palmmy2542) [inline src/Pages/Form/Components/ChiefComplaintSection.tsx]: Please extract all imports to default imports(eg `import Box from '@material-ui/core/Box'`). Because importing object is importing all of component in @material-ui/core.

File: package.json [hunk 8]
```diff
@@ -6,6 +6,7 @@
     "@date-io/date-fns": "^1.3.13",
     "@material-ui/core": "^4.12.1",
     "@material-ui/icons": "^4.11.2",
+    "@material-ui/lab": "^4.0.0-alpha.60",
```
Comment (palmmy2542) [inline package.json:9]: Why it should be loaded more libs?

File: src/Pages/Form/Components/SectionTitle.tsx [hunk 9]
```diff
@@ -8,7 +8,7 @@ const useStyle = makeStyles((theme: Theme) =>
   createStyles({
     root: {
       width: "100%",
```
Comment (palmmy2542) [inline src/Pages/Form/Components/SectionTitle.tsx]: I suggest to add marginBottom: 32px.

File: src/Pages/Form/Components/ChiefComplaintSection.tsx [hunk 10]
```diff
@@ -0,0 +1,207 @@
...
+      "& > *": {
+        margin: theme.spacing(3),
+      },
+    },
+    button: {
+      boxShadow: "0px 2px 4px rgba(0, 0, 0, 0.1)",
+      borderRadius: "20px",
+      width: "180px",
+      height: "81px",
+      right: "20px",
```
Comment (palmmy2542) [inline src/Pages/Form/Components/ChiefComplaintSection.tsx]: Please describe, why it should be set with right 20px?

File: package.json (hunk 8 above)
Comment (Wannakorn-C) [reply package.json:9]: Because I want to use ToggleButton and ToggleButtonGroup

**Review by palmmy2542 — APPROVED** [review]

**Review by palmmy2542 — APPROVED** [review]
